services/core-api/app/api/projects/project_summary/models/project_summary.py

```diff
@@ -457,6 +457,66 @@ def validate_project_party(cls, party, section):
         if not v.validate(party):
             return json.dumps(v.errors)
         return True
+    
+    @classmethod
+    def validate_mine_component_offsite_infrastructure(self, data):
```

Comment:
All the adjustments in this file was to move the validation of section specific properties out of the method that validates surface level properties and instead into the methods that actually validate the section these specific properties exist in.

So for example `facility_pid_pin_crown_file_no exists` on the surface level of the payload but on the front end it is in the Location, Access and Land Use (formerly labelled as Legal Landowner Information) section